fixmyapp/views.py

```diff
@@ -324,7 +324,19 @@ def post(self, request, campaign, pk, access_key):
         return Response(serialization)
 
 
+class ProfileDetail(generics.UpdateAPIView):
+    queryset = Profile.objects.all()
+    lookup_url_kwarg = "profile_id"
+    serializer_class = ProfileSerializer
+
+
+class ProfileCreate(generics.CreateAPIView):
+    queryset = Profile.objects.all()
+    serializer_class = ProfileSerializer
+
+
 @api_view(['PUT'])
+@permission_classes((permissions.AllowAny,))
```

Comment:
adding this decorator resolved the issue and made the classes above unnecessary

Reply:
I'm asking myself what the permissions are now ^^